mesh/pmesh.cpp

```diff
@@ -2414,13 +2419,15 @@ GetSharedFaceTransformations(int sf, bool fill2)
    FaceElemTr.Elem1No = face_info.Elem1No;
    GetElementTransformation(FaceElemTr.Elem1No, &Transformation);
    FaceElemTr.Elem1 = &Transformation;
+   mask += 1;
```

Comment:
I agree, I see the new `ConfigMasks`. Maybe you could put them to the declaration of `GetFaceElementTransformations` to point the user the right way. Maybe it will cause some warnings with existing code, but int should be implicitly convertible to enum. In the worst case, add a note to the documentation at least.